### ISSUE TITLE
Checking Bianchi identity

### ISSUE THREAD

**Issue by rdisipio** [einsteinpy/einsteinpy#537]
🐞 **What bugs you?**
Checking the [Bianchi identity](https://en.wikipedia.org/wiki/Contracted_Bianchi_identities) (see also [this](http://einsteinrelativelyeasy.com/index.php/general-relativity/69-bianchi-identity)) is a central piece of General Relativity. However, I'm not sure how to prove with EinsteinPy that a given (possibly arbitrary) metric fulfills such requirement. It would be nice if there was a pre-defined function to do that.

🎯 **Goal**

Make a standard tool to calculate the covariant derivatives of the Riemann tensor associated with a given metric tensor and to show that the cyclic application sums to zero.

💡 **Possible solutions**

I believe the basic pieces are there. If there are already functions to calculate the covariant derivatives and the Riemann tensor, it should be straightforward.

Thanks for any feedback!

**Comment by shreyasbapat** [comment]
Thanks for reporting this @rdisipio ! 
cc @JeS24 , we can discuss this in our next meeting :D

**Comment by gimseng** [comment]
This is a good idea. However, on the mathematical side, if the Riemann tensor is built out of a metric tensor, then the Rieman tensor automatically satisfies the Bianchi identity.

I think this should be a useful thing for a unit test type of thing. I forgot how EinsteinPy implements testing, is this something useful?

**Comment by JeS24** [comment]
@gimseng Precisely. Although, we can implement this now, let's wait till some progress has been made with #405, specifically with the covariant derivative operation.
